EIPS/eip-5521.md

```diff
@@ -25,7 +27,7 @@ By adding the `referring` indicator, users can mint new NFTs (e.g., C, D, E) by
 
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
 
```

Comment:
Your specification section doesn't include enough detail for an implementer to actually implement your standard. You should probably include a full Solidity interface here so readers are aware of function parameters and types, events, etc.

Reply:
> Your specification section doesn't include enough detail for an implementer to actually implement your standard. You should probably include a full Solidity interface here so readers are aware of function parameters and types, events, etc.

The interface spec's added now. Thanks.

Reply:
> Your specification section doesn't include enough detail for an implementer to actually implement your standard. You should probably include a full Solidity interface here so readers are aware of function parameters and types, events, etc.

The interface spec's added to the current version. Thank you.